dspace-api/src/main/java/org/dspace/core/Constants.java

```diff
@@ -91,6 +91,11 @@ public class Constants {
      */
     public static final String METADATA_BUNDLE_NAME = "METADATA";
 
+    /**
+     * Bundle name for a IIIF manifest
+     */
```

Comment:
This looks good. But I think there also needs to be a bit of work on the Angular side. When uploading a bitstream in the UI this bundle should only appear for items that are IIIF enabled. So in addition to updating the Angular configuration file there needs to be an extra check in the pulldown component for upload bundles.